pages/index.tsx

```diff
@@ -1,22 +1,53 @@
 import { useAuth0 } from '@auth0/auth0-react';
-import Link from 'next/link';
-import { ReactElement } from 'react';
-import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+import { ReactElement, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { RootState } from '../store';
+import { setToken } from '../store/actions/auth0';
+import { setUser } from '../store/actions/user';
 
 const Home = (): ReactElement => {
-  const { t } = useTranslation('pages');
-  const { user } = useAuth0();
+  const {
+    user: auth0User,
+    getAccessTokenSilently,
+  } = useAuth0();
+  const dispatch = useDispatch();
+  const router = useRouter();
+  const auth0 = useSelector((state: RootState) => state.auth0);
+  const user = useSelector((state: RootState) => state.user);
+
+  const getAccessToken = async () => {
+    try {
+      const accessToken = await getAccessTokenSilently({
+        audience: `https://${process.env.AUTH0_DOMAIN}/api/v2/`,
```

Comment:
Question of an ignoramus, but one who has recently run into an adjacent problem: Are you using a preprocessor to turn this into a static resource on compile? I know create-react-app supports REACT_APP-prefixed env variables, but I'm assuming this is something slightly more sophisticated.

Reply:
I believe that the compilation is done via Next.js, though you're right in that these might need a prefix. These might need to be prefixed with `NEXT_PUBLIC_` (per [_Exposing Environment Variables to the Browser_](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser)).

Reply:
Yeah looking at the resource @whymarrh shared I can probably refactor these into a `.env` file at the very least. Not sure if prefixes are necessary as they are working as-is.

Reply:
Fixed in #8c879ac